homeassistant/components/sensor/device_trigger.py

```diff
@@ -251,7 +254,7 @@ async def async_get_triggers(
                 **automation,
                 "platform": "device",
                 "device_id": device_id,
-                "entity_id": entry.entity_id,
+                "entity_id": entry.id,
```

Comment:
This is confusing.

Is there some discussion why we think this is a good idea?